docs/platforms/python/integrations/langchain/index.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Langchain
+description: "Learn about using Sentry for Langchain."
+---
+
+This integration connects Sentry with [Langchain](https://github.com/langchain-ai/langchain).
+The integration has been confirmed to work with Langchain 0.1.11.
+
+## Install
+
+Install `sentry-sdk` from PyPI and the appropriate langchain packages:
+
+```bash
+pip install --upgrade 'sentry-sdk' 'langchain-openai' 'langchain-core'
+```
+
+## Configure
+
+If you have the `langchain` package in your dependencies, the Langchain integration will be enabled automatically when you initialize the Sentry SDK.
+
+An additional dependency, `tiktoken`, is required to be installed if you want to calculate token usage for streaming chat responses.
+
+<SignInNote />
+
+```python
+from langchain_openai import ChatOpenAI
+import sentry_sdk
+
+sentry_sdk.init(
+    dsn="___PUBLIC_DSN___",
+    enable_tracing=True,
+    traces_sample_rate=1.0,
+    send_default_pii=True, # send personally-identifiable information like LLM responses to sentry
+)
+
+llm = ChatOpenAI(model="gpt-3.5-turbo-0125", temperature=0)
+```
+
+## Verify
+
+Verify that the integration works by inducing an error:
+
+```python
+from langchain_openai import ChatOpenAI
+import sentry_sdk
+
+sentry_sdk.init(...)  # same as above
+
+llm = ChatOpenAI(model="gpt-3.5-turbo-0125", temperature=0, api_key="bad API key")
+with sentry_sdk.start_transaction(op="ai-inference", name="The result of the AI inference"):
+    response = llm.invoke([("system", "What is the capital of paris?")])
+    print(response)
+```
+
+After running this script, a transaction will be created in the Performance section of [sentry.io](https://sentry.io). Additionally, an error event (about the bad API key) will be sent to [sentry.io](https://sentry.io) and will be connected to the transaction.
+
+It may take a couple of moments for the data to appear in [sentry.io](https://sentry.io).
+
+## Behavior
+
+- The Langchain integration will connect Sentry with Langchain and automatically monitor all LLM, tool, and function calls.
+
+- All exceptions in the execution of the chain are reported.
+
+- Sentry by default considers LLM and tokenizer inputs/outputs as PII. If you want to include them, set `send_default_pii=True` in the `sentry_sdk.init()` call. To explicitly exclude prompts and outputs despite `send_default_pii=True`, configure the integration with `include_prompts=False` as shown in the [Options section](#options) below.
```

Comment:
```suggestion
- Sentry considers LLM and tokenizer inputs/outputs as PII and, by default, does not include PII data. If you want include the data, then set `send_default_pii=True` in the `sentry_sdk.init()` call. To explicitly exclude prompts and outputs despite `send_default_pii=True`, configure the integration with `include_prompts=False` as shown in the [Options section](#options) below.
```
I think when we say "by default" here, we're actually talking about the default behavior of how we treat PII (it's not included/sent), and that's the default behavior we're changing (sending the PII). 

I think the word "include" is also vague. Would it be more accurate to talk about this as data scrubbing? e.g. "Sentry considers LLM and tokenizer inputs/outputs as PII and, by default, scrubs PII data. If you want to send the data unscrubbed, then set `send_default_pii=True` in the `sentry_sdk.init()` call."